examples/storybook/src/wrap-with-hits.ts

```diff
@@ -0,0 +1,139 @@
+import { Component } from '@angular/core';
+
+type Helper = {
+  search: Function;
+  addDisjunctiveFacetRefinement: Function;
+};
+
+export function wrapWithHits({
```

Comment:
This helper function was copied almost as is from dev-novel 
https://github.com/algolia/angular-instantsearch/blob/develop/examples/dev-novel/wrap-with-hits.ts